web/src/pages/Address/index.tsx

```diff
@@ -99,27 +104,31 @@ export default (props: React.PropsWithoutRef<RouteComponentProps<{ address: stri
     })
   }
 
-  const getTransactions = (page: number, size: number) => {
-    fetchTransactionsByAddress(address).then(response => {
+  const getTransactions = (page_p: number, size_p: number) => {
+    fetchTransactionsByAddress(address, page_p, size_p).then(response => {
       const { data, meta } = response as Response<TransactionWrapper[]>
       if (meta) {
         const { total } = meta
         setTotalTransactions(total)
       }
-      const transactions = data.slice((page - 1) * size, page * size)
+      const transactions = data.slice((page_p - 1) * size_p, page_p * size_p)
       setTrasactionWrappers(transactions)
     })
   }
 
   useEffect(() => {
     getAddressInfo()
-    getTransactions(pageNo, pageSize)
-  }, [])
+    const page_p = page ? parseInt(page as string, 10) : pageNo
+    const size_p = size ? parseInt(size as string, 10) : pageSize
+    setPageNo(page_p)
+    setPageSize(size_p)
+    getTransactions(page_p, size_p)
+  }, [pageNo, pageSize])
```

Comment:
it is better to listen the change of the props.location.search , or it will not trigger update while browser goback

Reply:
Get it.